tests/integration/delete.bats

```diff
@@ -51,3 +51,49 @@ function teardown() {
   runc delete --force notexists
   [ "$status" -eq 0 ]
 }
+
+@test "runc delete --force in cgroupv2 with subcgroups" {
+  requires cgroups_v2 root
+  set_cgroups_path "$BUSYBOX_BUNDLE"
+
+  # grant `rw` priviledge to `/sys/fs/cgroup`
+  DATA=$(cat <<EOF
+    "destination": "/sys/fs/cgroup",
+    "type": "cgroup",
+    "source": "cgroup",
+    "options": [
+            "nosuid",
+            "noexec",
+            "nodev",
+            "relatime",
+            "rw"
+    ]
+EOF
+    )
+  DATA=$(echo ${DATA} | sed 's/\n/\\n/g' | sed 's/\//\\\//g')
+  sed -i ":a;N;s/\n//g;ta" ${BUSYBOX_BUNDLE}/config.json
+  sed -i "s/\"destination\": \"\/sys\/fs\/cgroup\",[^}]*/${DATA}/" ${BUSYBOX_BUNDLE}/config.json
```

Comment:
maybe you can use `jq` here instead of `sed`, could be more readable.

Reply:
In fact @AkihiroSuda already added a helper function that does it, see https://github.com/opencontainers/runc/pull/2416/